### PR TITLE
chore: Snyk Ignore File

### PR DESCRIPTION
**Pull request by otaviof**
Creating a `.snyk` file to describe ignored project issues.

### CODE REVIEW

File: .snyk [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+ignore:
+  5ed8b131-b1a4-4c9f-a6b4-75eecf2396b7:
+    - "vendor/github.com/docker/distribution/registry/api/v2/urls.go":
+      reason: |
+        SSRF: The URL is static, coming from the "${DOCKERCONFIG}" file, thus
+        while the URL is configurable it's not dynamic.
+      expires: "2025-01-01"
```
Comment (Roming22) [inline .snyk]: nit pick: I don't think we should choose a date that falls on a non-working day.

**Comment by Roming22** [comment]
@ellin-rh Can you please review this PR?

**Comment by ellin-rh** [comment]
@Roming22 @otaviof agree w Romain's nitpick comment. Otherwise, lgtm! Verified ID's and formatting. Thank you for the changes!

File: .snyk (hunk 1 above)
Comment (otaviof) [reply .snyk]: Sure! Good point.

**Comment by otaviof** [comment]
> @Roming22 @otaviof agree w Romain's nitpick comment. Otherwise, lgtm! Verified ID's and formatting. Thank you for the changes!

Thank you, folks. Please consider the latest changes.

**Pull request merged**